pipeline-adapters/mlrun-pipelines-kfp-v2/src/mlrun_pipelines/ops.py

```diff
@@ -143,6 +145,24 @@ def add_function_node_selection_attributes(
     return task
 
 
+def _enrich_node_selector_from_project(function):
```

Comment:
can be moved to kfp-common.helpers

Reply:
There is no `import mlrun` in the helpers, and I don't think it's a good idea to add it just for this function. Also, I think it fits better in common.ops anyway, so I have moved it there.